meta-resin-common/recipes-connectivity/bluez5/bluez5.bbappend

```diff
@@ -0,0 +1,13 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SRC_URI += " \
+    file://10-local.rules \
+    file://hciconf.sh \
+    "
+
+do_install_append() {
+    install -d  ${D}/etc/bluetooth/
+    install -m 0755 ${WORKDIR}//hciconf.sh ${D}/etc/bluetooth/
```

Comment:
You can use -D and not need to create the directory anymore as a separate command.
As well you have double slash in the path.